api/src/main/java/org/apache/iceberg/actions/RewriteDataFiles.java

```diff
@@ -119,6 +119,16 @@ public interface RewriteDataFiles
 
   String REWRITE_JOB_ORDER_DEFAULT = RewriteJobOrder.NONE.orderName();
 
+  /**
+   * Constant representing the target output partition specification ID.
```

Comment:
nit: I don't think this is a constant

Perhaps just, "The partition specification ID to be used for rewritten files"